src/markdown/tutorial/part-1/03-automated-testing.md

```diff
@@ -103,7 +103,7 @@ Finally, we asserted that clicking on the link should bring us to the `/about` U
 > Here, we are writing the tests in a framework called QUnit, which is where the functions `module`, `test` and `assert` come from. We also have additional helpers like `click`, `visit`, and `currentURL` provided by the `@ember/test-helpers` package. You can tell what comes from which package based on the `import` paths at the top of the file. Knowing this will be helpful when you need to search for documentation on the Internet or ask for help.
 
 ```run:command hidden=true cwd=super-rentals
-ember test --path dist
+pnpm test
 git add tests/acceptance/super-rentals-test.js
 ```
```

Comment:
Below this discusses using `ember test -s` ... I generated an app as the tutorial does and ran `ember test -s` and it errors with: `Not found: /1179115527583250/tests/index.html?hidepassed`

Reply:
regardless of what we do when people run `ember t -s`, I've updated the passage to tell people to just run `npm start` and naviate to `/tests` 👍